_posts/2022-11-08-dotnet7_ga.md

```diff
@@ -0,0 +1,27 @@
+---
+title: ".NET 7 GA available on App Service"
+author_name: "Byron Tardif"
+---
+
+We have complted the initial rollout for [.NET 7 GA](https://dotnet.microsoft.com/download/dotnet/7.0) support on App Service. Like in previous years we are using the [App Service Early Access feature](https://aka.ms/app-service-early-access) to enable day-0 support on the platform across all public regions on both Windows and Linux App Service Plans. The early access release will be followed by additional deployments to fully integrate the new bits across our fleet expecting to be fully done by the end of the week.
+
+If you already have an app targeting and earlier preview of .NET 7.0 on the platform, there is no need to take action as the new runtime will be picked up on the next application restart once the update is available for your app. You can trigger this manually by starting and stopping your app.
+
+Self-contained .NET apps will *not* be auto-updated since they have no dependency on the runtime provided by App Service.
+
+[Azure Functions](https://go.microsoft.com/fwlink/?linkid=2214834) and [Azure Static Webapps](https://go.microsoft.com/fwlink/?linkid=2215233) are also enabling .NET 7 workloads across their scenarios.
+
+
+If you want to learn more, be sure to checkout our session during [.NET Conf 2022](https://www.dotnetconf.net/agenda):
+
+- Tuesday 11/09 @ 11:00am PST Join [Scott Hunter](https://twitter.com/coolcsh) to talk about the latest happening in the world of building cloud native applications with .NET and Azure
+- Thursday 11/10 @ 1:00am PST Join [@bktv99](https://twitter.com/bktv99) and [@segaurav](https://twitter.com/segaurav) will be taking the stage talk about  ".NET 7.0 and bulk migration of ASP.Net web apps to App Services".
+
+You can also follow us on twitter for more updates and news: [@AzAppService](https://twitter.com/AzAppService/)
+
+Next steps:
+
+- [Download .NET 7](https://dotnet.microsoft.com/download/dotnet/7.0)
+- [Anouncing .NET 7.0 ]()
+- [ASP.NET Core in .NET 7]()

```

Comment:
missing go links?

Reply:
yeah will add this one they go live tomorrow